modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -1,7 +1,44 @@
-[[kubernetes]]
-= VHM and SUSE Container-as-a-Service Platform
+[[vhm-caasp]]
+= VHM and SUSE CaaS Platform
 
-You can use a {productname} VHM to gather instances from SUSE Container-as-a-Service Platform ({caasp}).
+You can use a {productname} VHM to gather instances from {caasp}.
 
 The VHM allows {productname} to obtain and report information about your virtual machines.
 For more information on VHMs, see xref:client-configuration:vhm.adoc[].
+
+
+
+== Register CaaSP nodes
+
+You can register each {caasp} node to {productname} using the same method as you would a Salt client.
+For more information, see xref:client-configuration:registration-overview.adoc[].
+
+We recommend that you create an activation key to associate {caasp} channels, and to onboard the associated nodes.
+For more on activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[]
+
+If you are using ``cloud-init``, we recommended that you use a bootstrap script in the ``cloud-init`` configuration.
+For more on bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
+
+
+When you have added the {caasp} nodes to {productname}, you will be able to:
+
+* View the patch level status of each node
+* Perform configuration management operations against the nodes, including using Salt states.
+* Assign different set of channels to clusters using channel staging and lifecycle management filters.
+
+Additionally, updates related to {k8s} are automatically installed, because of a self-updating daemon running on the {caasp} nodes.
+// Is this referring to skuba? LKB 2019-11-11
+For more information, see the {caasp} documentation.
```

Comment:
Maybe a reference to our Admin Guide would be helpful here: https://susedoc.github.io/doc-caasp/master/caasp-admin/single-html/#_cluster_updates

Reply:
Yes, I agree. Just wanted to find out exactly what it's referring to first :wink: